### ISSUE TITLE
City Council Spider Times are Incorrect

### ISSUE THREAD

**Issue by diaholliday** [City-Bureau/documenters-aggregator#221]
Not sure if this applies other spiders or not but the City Council start times are off by several hours. For example, I know the committee on education starts at 2:30p tomorrow, not 8:30p. Is there a way to correct this and, second priority, any others that may be off?
![screen shot 2018-01-11 at 1 13 48 pm](https://user-images.githubusercontent.com/4017450/34842583-8c1f8b6c-f6d1-11e7-8073-9bd863c5c500.png)

**Comment by easherma** [comment]
Sounds like it's probably a UTC time error given the hour interval. Should be an easy fix, I might be able to address this today and fix it on the scraper end. As for the data already in Airtable, I don't have access at the moment so I can't do much there.

**Comment by diaholliday** [comment]
I'll defer to @r-wei on the push to Airtable, she has access (the accounts cost $$ so I'm trying to keep the number of accounts down until we have a better sense of who needs access and/or if a limited number of people can just push all necessary changes there)

**Comment by easherma** [comment]
The spider appears to be outputting start times as I'd expect ( i.e. 2018-01-11T16:00:00-06:00 is 10:00 AM in Chicago which is right for https://chicago.legistar.com/MeetingDetail.aspx?ID=584299&GUID=C817555E-7ABB-4FAF-B79E-5CD1C4630278&Options=info&Search= ), so the issue may be in the way it's sent to Airtable, or the way Airtable is interpreting it. 

I'd wager the last one is probably the problem.

**Comment by easherma** [comment]
It sounds like Airtable itself may be defaulting to a timezone, maybe there
is a project or account setting with a timezone selected?

**Comment by r-wei** [comment]
I can fix this. I think 2018-01-11T16:00:00-06:00 is 4pm Chicago time, so it's the spider not Airtable. The start times for the ward nights are showing up correctly in Airtable.

**Comment by easherma** [comment]
Isn't 16:00 - 6 10 AM? I checked the spider output locally but this seemed
to line up to me.

**Comment by r-wei** [comment]
```
>>> import datetime
>>> import pytz
>>> datetime.datetime.now()
datetime.datetime(2018, 1, 12, 16, 49, 48, 404872)
>>> chicago_tz = pytz.timezone('America/Chicago')
>>> chicago_tz.localize(datetime.datetime.now()).isoformat()
'2018-01-12T16:50:46.670077-06:00'
```

I did this just now and it's 4:50pm... which suggests to me that "16:00...-06:00" means 4pm Chicago time. Does that seem right to you?

**Comment by r-wei** [comment]
resolved by #223
